src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -139,6 +139,8 @@ RoverPositionControl::manual_control_setpoint_poll()
 						_reset_yaw_sp = false;
 
 					} else {
+						// TODO: Clarify relationship between RATE_MAX and MAN_Y_MAX.
+						// Currently, in Acro mode we use RATE_MAX as max rate setpoint, and in manual (attitude mode == stabilized), MAN_Y_MAX.
```

Comment:
We currently have 2 different yaw rate maximum limits, but do we really need two? I think we can use only `GND_RATE_MAX`, which should be dependent on the vehicle's maneuverability, and remove manual control specific max rate param.